src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyboardButtonClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyboardButtonClick);
```

Comment:
I think you can give a simple name
Up to you
```suggestion
    document.removeEventListener('keyup', this.handleClick);
```